custom-recipes/neo4j-export-relationships/recipe.json

```diff
@@ -141,41 +141,70 @@
             "description": "",
             "visibilityCondition": "model.property_names_mapping"
         },
+        {
+            "name": "expert_mode",
+            "label": "Expert mode",
+            "type": "BOOLEAN",
+            "description": "",
+            "defaultValue": false
+        },
         {
             "label": "Extra properties",
-            "type": "SEPARATOR"
+            "type": "SEPARATOR",
+            "visibilityCondition": "model.expert_mode"
         },
         {
             "name": "node_count_property",
             "label": "Node count",
             "type": "BOOLEAN",
             "description": "When a node is updated, its 'count' property is incremented.",
-            "defaultValue": false
+            "defaultValue": false,
+            "visibilityCondition": "model.expert_mode"
         },
         {
             "name": "edge_weight_property",
             "label": "Relationship weight",
             "type": "BOOLEAN",
             "description": "When a relationship is updated, its 'weight' property is incremented.",
-            "defaultValue": false
+            "defaultValue": false,
+            "visibilityCondition": "model.expert_mode"
         },
         {
             "label": "Export parameters",
-            "type": "SEPARATOR"
+            "type": "SEPARATOR",
+            "visibilityCondition": "model.expert_mode"
         },
         {
             "name": "load_from_csv",
             "label": "Import data from CSV files",
             "type": "BOOLEAN",
-            "description": "Requires output folder to be in a SCP/SFTP connection to the Neo4j import directory.",
-            "defaultValue": false
+            "description": "Requires output folder to be in a connection pointing to the Neo4j import directory.",
+            "defaultValue": false,
+            "visibilityCondition": "model.expert_mode"
+        },
+        {
+            "name": "batch_size",
+            "label": "Batch size",
+            "type": "INT",
+            "description": "Number of rows to send in each Cypher queries.",
+            "defaultValue": 500,
```

Comment:
You could use a `"minI": 1` for here and `csv_size`, and same for the export node json